src/planscape/plan/views.py

```diff
@@ -270,3 +280,50 @@ def get_project_areas(request: HttpRequest) -> HttpResponse:
         return JsonResponse(response)
     except Exception as e:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
+
+
+def get_scores(request: HttpRequest) -> HttpResponse:
+    try:
+        plan = get_plan_by_id(request.GET)[0]
+        geo = plan.geometry
+        reg = plan.region_name.removeprefix('RegionName.').lower()
+
+        accumulator = RasterPixelAccumulator(geo)
+
+        ids_to_conditions = {
+            c.id: c.condition_name
+            for c in BaseCondition.objects.filter(region_name=reg).all()}
+        ids_to_raster_names = {
+            c.condition_dataset_id: c.raster_name
+            for c in Condition.objects.filter(is_raw=False).all()}
+        filtered_raster_names_to_ids = {
+            ids_to_raster_names[id]: id for id in ids_to_raster_names
+            if id in ids_to_conditions}
+
+        for r in ConditionRaster.objects.filter(
+                name__in=filtered_raster_names_to_ids.keys()).filter(
+                raster__bboverlaps=geo).all():
+
+            if r.raster is None:
+                continue
+            id = filtered_raster_names_to_ids[r.name]
+            condition = ids_to_conditions[id]
+            accumulator.process_raster(
+                r.raster, condition)
+
+        conditions = []
+        for c in accumulator.stats.keys():
+            if accumulator.stats[c]['count'] == 0:
+                conditions.append({'condition': c})
+            else:
+                conditions.append(
+                    {'condition': c, 'mean_score': accumulator.stats[c]
+                     ['sum'] / accumulator.stats[c]['count']})
+
+        response = {'conditions': json.dumps(conditions)}
```

Comment:
I'm testing this on the frontend now and the field is coming across as a string blob instead of JSON:

![image](https://user-images.githubusercontent.com/10444733/213589687-ccc14af2-ca8f-4e2a-97f0-699d8059da8f.png)

Maybe remove the `json.dumps`?

Reply:
noted - I've made this change in #383.